activiti-spring-boot/spring-boot-starters/activiti-spring-boot-audit-starter/.gitignore

```diff
@@ -0,0 +1,13 @@
+/target
```

Comment:
Let's delete this file as it's not adding any new exclusion that's not in the root .gitignore file.